manifests/i/IngoHdeBoer/WinShell/4.0.0.6/IngoHdeBoer.WinShell.installer.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: IngoHdeBoer.WinShell
+PackageVersion: 4.0.0.6
+Installers:
+- InstallerUrl: https://www.winshell.org/WinShellSetup.exe
+  Architecture: x64
+  InstallerType: burn
+  InstallerSha256: 4FAAF14AE77E5E154BAC65D93451DDD163ABE18DEBBC62CD34011A700F2A0E13

```

```suggestion
  InstallerSha256: 4FAAF14AE77E5E154BAC65D93451DDD163ABE18DEBBC62CD34011A700F2A0E13
  ProductCode: '{dc994bff-9ceb-4f8f-8f7b-12a53df0a2b5}'
```
This app writes two entries to ARP. The product code is needed to differentiate them